conda_forge_tick/os_utils.py

```diff
@@ -84,14 +85,16 @@ def clean_disk_space(ci_service: str = "github-actions") -> None:
   ; do
     sudo rsync --stats -a --delete /opt/empty_dir/ $d || true
   done
-  sudo apt-get purge -y -f firefox \
+  # dpkg does not fail if the package is not installed
+  sudo dpkg --remove -y -f firefox \
                           google-chrome-stable \
                           microsoft-edge-stable
   sudo apt-get autoremove -y >& /dev/null
   sudo apt-get autoclean -y >& /dev/null
   sudo docker image prune --all --force
   df -h
-""")
+"""
+                )
```

Comment:
This is weird formatting. I much prefer the idiom:

```python
f.write(
    dedent(
        """
        whatever
        whatever
        """
    ).lstrip()
)
```

or `inspect.cleandoc()`.

Just a note in case this was intentional and not just formatting noise.

Reply:
This one looks like ruff-format running against the old code as opposed to a new change. I agree with @jaimergp but I am also ok if we defer this to another PR to go through and use dedent more widely.

Reply:
Yeah let's go through this in a separate PR.